client/reader/following-stream/post-unavailable.jsx

```diff
@@ -16,13 +16,13 @@ var PostUnavailable = React.createClass( {
 
 	componentWillMount: function() {
 		this.errors = {
-			403: this.translate( 'This is a private site. You’re following the site, but not currently a member. Please request membership to display these posts in Reader.' ),
+			'unauthorized': this.translate( 'This is a private site. You’re following the site, but not currently a member. Please request membership to display these posts in Reader.' ),
```

Comment:
While we're here, maybe change this to "this post" from "these posts" since the error applies to one post?